steps/steps.go

```diff
@@ -45,6 +51,23 @@ func DefaultPrepareStepList(isIncludeCache bool) []bitriseModels.StepListItemMod
 	return append(stepList, ScriptSteplistItem(ScriptDefaultTitle))
 }
 
+// DefaultPrepareStepListV2 ...
+func DefaultPrepareStepListV2(params PrepareListParams) []bitriseModels.StepListItemModel {
```

Comment:
This comment applies to the `DefaultDeployStepListV2` function too. What if we modify the original functions instead and let all of the workflows benefit from them?

Putting the deploy step last will decrease the failure rate for all of them. And it is true that SSH key activation skipping will require a little extra work from the scanners and immediate benefit. 

So I think I would change the deploy one for all of the workflow and the other is up for debate. What do you think?

Reply:
I would prefer to apply the change in smaller increments, as it also requires manually editing the e2e tests.